### PR TITLE
Add number of installations that are reporting

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: worker/src/handlers/schedule.ts [hunk 1]
```diff
@@ -90,11 +93,14 @@ const generateCurrentDataset = (
       count_users.push(huuid.user_count);
     }
 
-    for (const integration of huuid.integrations || []) {
-      if (!integrations[integration]) {
-        integrations[integration] = 1;
-      } else {
-        integrations[integration]++;
+    if (reported_integrations.length === 0) {
```
Comment (bramkragten) [inline worker/src/handlers/schedule.ts:96]:
```suggestion
    if (reported_integrations.length) {
```

**Comment by bramkragten** [comment]
Should we do the same for the average users/automations/etc... charts?

**Comment by ludeeus** [comment]
Make sense.

**Review by bramkragten — APPROVED** [review]

File: site/src/analytics-integrations.ts [hunk 2]
```diff
@@ -168,6 +168,10 @@ export class AnalyticsIntegrations extends LitElement {
           </svg>
         </mwc-icon-button>
       </div>
+      <div class="footer">
+        ${this.lastDataEntry!.reports_integrations || "Unkown"} installations is
```
Comment (bramkragten) [inline site/src/analytics-integrations.ts:172]: are

**Review by bramkragten — APPROVED** [review]

**Pull request merged**